src/components/rendering/renderable.rs

```diff
@@ -3,21 +3,25 @@ use components::rendering::Texture;
 use ecs::{Component, VecStorage};
 
 #[derive(Clone)]
-/// This struct is a `Component`, which combines geometry and ka, kd textures.
+/// This struct is a `Component`, which combines geometry and ka/kd/ks textures.
 /// Every `Renderable`, `Transform` pair attached to an entity inside the `World`
 /// is rendered by `GfxDevice::render_world` method.
 pub struct Renderable {
     pub mesh: Mesh,
     pub ka: Texture,
     pub kd: Texture,
+    pub ks: Texture,
+    pub ns: f32,
```

Comment:
These names are kinda confusing, maybe something longer would be better and documentation.